API/Controllers/OrdersController.cs

```diff
@@ -34,21 +34,54 @@ public OrdersController(IUnitOfWork unitOfWork)
         }
 
         [HttpGet]
+        [Authorize(Roles = "Administrator,Worker,Client")]
         public async Task<ActionResult> GetOrders()
         {
-            // admin wszystkie
-            // worker wolne do wziecia lub przypisane do siebie 
-            // klient tylko swoje
-            return Ok();
+            if (User.IsInRole("Worker"))
+                return Ok(await ordersRepo.GetAll(filter:x=>x.EmployeeId==User.GetId() || x.OrderStatus.OrderStatusId==1,
+                                                  orderBy:x=>x.OrderByDescending(x=>x.ServiceDate),
+                                                 includes:x => x.Include(x => x.OrderStatus)
+                                                                .Include(x => x.Employee)));
+            else if (User.IsInRole("Client"))
+                return Ok(await ordersRepo.GetAll(filter:  x => x.ClientId == User.GetId(),
+                                                  orderBy: x => x.OrderByDescending(x => x.ServiceDate),
+                                                 includes: x => x.Include(x => x.OrderStatus)
+                                                                 .Include(x => x.Employee)));      
+            else if (User.IsInRole("Administrator"))
+                return Ok(await ordersRepo.GetAll(
+                     orderBy: x => x.OrderByDescending(x => x.ServiceDate),
+                    includes: x => x.Include(x => x.OrderStatus)
+                                    .Include(x => x.Employee)));
+            else
```

Comment:
jak mamy atrybut authorize to w ogóle nie musimy sprawdzać czy user jest authorized.